core/tests/protractor_utils/ExplorationEditorMainTab.js

```diff
@@ -801,9 +801,9 @@ var ExplorationEditorMainTab = function() {
     // Return as-is if string does not contain placeholders.
     if (placeholders) {
       // Replacing placeholders in ruleDescription with given text.
-      placeholders.forEach(function(placeholderElement, index) {
+      for (var index = 0; index < placeholders.length; ++index) {
```

Comment:
```suggestion
      for (var index = 0; index < placeholders.length; index++) {
```

It doesn't affect the logic, but `index++` is more canonical I think